src/main/java/frc/trigon/robot/subsystems/climber/placeholderclimber/PlaceholderClimberConstants.java

```diff
@@ -0,0 +1,95 @@
+package frc.trigon.robot.subsystems.climber.placeholderclimber;
+
+import com.ctre.phoenix6.StatusSignal;
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.mechanisms.DifferentialMechanism;
+import com.ctre.phoenix6.signals.InvertedValue;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+import frc.trigon.robot.constants.RobotConstants;
+
+public class PlaceholderClimberConstants {
+    static final boolean ENABLE_FOC = true;
+    private static final int
+            RIGHT_MOTOR_ID = 0,
+            LEFT_MOTOR_ID = 0;
+    private static final InvertedValue
+            RIGHT_MOTOR_INVERTED_VALUE = InvertedValue.CounterClockwise_Positive,
+            LEFT_MOTOR_INVERTED_VALUE = InvertedValue.CounterClockwise_Positive;
+    private static final NeutralModeValue NEUTRAL_MODE_VALUE = NeutralModeValue.Coast;
+    private static final boolean ARE_MOTORS_ALIGNED = true;
+    private static final double GEAR_RATIO = 1;
```

Comment:
use 1 gear ratio for the left and right motors, and for the sims and real io implementations